api/scripts/wazuh_apid.py

```diff
@@ -260,65 +280,34 @@ def error(self, msg, *args, **kws):
     logging.Logger.error = error
 
 
-if __name__ == '__main__':
+def get_script_arguments() -> Namespace:
+    """Get script arguments.
+
+    Returns
+    -------
+    argparse.Namespace
+        Arguments passed to the script.
+    """
+    parser = ArgumentParser()
+    parser.add_argument('-d', action='store_true', dest='daemon', help='Run as a daemon.')
+    parser.add_argument('-r', action='store_true', dest='root', help='Run as root')
```

Comment:
```suggestion
    parser.add_argument('-r', '--root', action='store_true', dest='root', help='Run as root')
```
Nevertheless, we should analyze why we would need to run the API or any other service as root.

Reply:
I agree, but given it's currently required to run the services I didn't remove it